src/sentry/integrations/vercel/integration.py

```diff
@@ -253,7 +253,7 @@ def create_env_var(self, client, vercel_project_id, key, value, type):
         data = {
             "key": key,
             "value": value,
-            "target": ["production"],
+            "target": ["production", "preview", "development"],
```

Comment:
Could we make this a constant or ENUM?

Reply:
To be honest, this is the only place where we need it in this file and I think it is okay to make it explicit like this.

We could, of course, make a constant out of this, and then share this constant with our test in `test_integration.py` but :) :
1. I don't know Python that much
2. I don't know if that is a good practice to assert against a variable that is constant in code